Modules/src/UserInputValidator.cc

```diff
@@ -16,16 +18,31 @@ namespace ChimeraTK {
 
   bool UserInputValidator::validate(const ChimeraTK::TransferElementID& change) {
     if(!change.isValid()) {
+      enableDeepValidation();
```

Comment:
I think it is/ was so far ok to call `validateAll()` directly instead of `validate({})`, so maybe better move the call to `enableDeepValidation()` into `validateAll()`.

Reply:
Changed